packages/openapi-generator/src/options/generator-options.spec.ts

```diff
@@ -0,0 +1,60 @@
+import { parseGeneratorOptions } from './generator-options';
+
+describe('parseGeneratorOptions', () => {
+  it('gets default options', () => {
+    expect(
+      parseGeneratorOptions({
+        input: 'inputDir',
+        outputDir: 'outputDir'
+      })
+    ).toEqual({
+      input: `${process.cwd()}/inputDir`,
+      outputDir: `${process.cwd()}/outputDir`,
+      transpile: false,
+      include: undefined,
+      clearOutputDir: false,
+      strictNaming: true,
+      tsConfig: undefined,
+      packageJson: false,
+      perServiceConfig: undefined,
+      packageVersion: '1.0.0',
+      readme: false
+    });
+  });
+
+  it('parses per service config file for file path', () => {
+    expect(
+      parseGeneratorOptions({
+        input: 'inputDir',
+        outputDir: 'outputDir',
+        perServiceConfig: 'someDir/config.json'
+      })
+    ).toMatchObject({
+      perServiceConfig: `${process.cwd()}/someDir/config.json`
+    });
+  });
+
+  it('parses per service config file for directory path', () => {
```

Comment:
Should we also have similar tests (fill path/dir path) but without `perServiceConfig`?

Reply:
Not sure, I understand this: You mean to check the default? Or other options?

Reply:
Yes, the purpose is to check the default behaviour with the two cases: file path and dir path.

Reply:
Added a few tests for extended behavior of the perserviceconfig.